FF1Lib/FF1Rom.cs

```diff
@@ -102,7 +103,23 @@ public static async Task<FF1Rom> CreateAsync(Stream readStream)
 
 			return rom;
 		}
+		public class RandomizerDigest
+		{
+			//This JSON gets embeded into the rom, so the property values have been
+			//shortened to save space
+			public String s { get; set; } //Randomizer Seed
```

Comment:
If you look at this link: https://www.newtonsoft.com/json/help/html/jsonpropertyname.htm
There is a simple way to make the PropertyName in teh code and the Name in the json differ.

I'm not pushing for a change here. With the comments it's totally fine by me. But I just wanted to let you know in case you need it sometime.

Reply:
This is more elegant, I like it!

Reply:
ok so while I like this, in the middle of trying to make this change visual studio crashed and now I can build the application anymore cause of this
![image](https://user-images.githubusercontent.com/6075260/142953593-7fc7675b-13df-409c-9247-19f720f7298f.png)

I reverted the property changes because I simply cannoy test them at this time

Reply:
got it to work!